bin/disco_deploy.py

```diff
@@ -28,6 +29,7 @@
      --hostclass HOSTCLASS  Limit command to a specific hostclass
      --environment ENV      Environment to operate in
      --allow-any-hostclass  Do not limit command to hostclasses defined in pipeline
+     --strategy STRATEGY    The deployment strategy to use. Currently supported: 'classic' or 'blue_green'.
```

Comment:
You can, but you shouldn't.

The way I've written the parsing logic in `test_ami`, anything passed in will override the options in `disco_aws.ini`. Because of that, providing a default here will override any other configuration, which is definitely not desired.

The fact that this overrides configuration is `disco_aws.ini` could be stated in the help text.